browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -172,34 +171,50 @@ BraveBrowserView::BraveBrowserView(std::unique_ptr<Browser> browser)
                           base::Unretained(this)));
 #endif
 
-#if BUILDFLAG(ENABLE_SIDEBAR)
+  const bool can_have_vertical_tabs =
```

Comment:
nit: `can_have` makes it sound like it's just an option, but you're checking `ShouldShowVerticalTabs`. Maybe `show_vertical_tabs`? Same for sidebar